docs/source/_static/theme_overrides.css

```diff
@@ -21,12 +21,18 @@
 /* Customizing with theme CSS variables */
 
 :root {
-  /* Change header hight to make the logo a bit larger */
-  --pst-header-height: 6rem;
   /* Make headings more bold */
   --pst-font-weight-heading: 600;
 }
 
+/* Change header hight to make the logo a bit larger */
+/* only on wider screens */
+@media only screen and (min-width: 950px){
```

Comment:
How did you decide on the min width of 950? From doing a quick experiment with the "Responsive design mode" in Firefox, I already see it wraparound (causing a very large header) at a width of around 1380 px

Reply:
Or this is only for the "mobile" layout? (where there is no header at all anymore, except for the logo and expand buttons)

I indeed suggested on the issue to do this in this case, but, we can maybe also already do it when you are on a normal but small screen, i.e. for the cases where all the header navbar items are visible, but start to wrap around because there is not enough space, like:

![image](https://github.com/apache/arrow/assets/1020496/87f49bc0-4934-4103-89a2-cc2fb130c991)

Reply:
I am not sure this will help any as the version switcher and search buttons are already causing issues (see https://github.com/pydata/pydata-sphinx-theme/issues/1493#issuecomment-1746819706), but am trying locally now to see.

Reply:
Ok, setting the minimum width of the screen to have the bigger header height to 1380px gave me two layouts to compare as the limit for the double header is a bit higher:

- just under 1380px and smaller header height:

![Screenshot 2023-10-09 at 11 08 31](https://github.com/apache/arrow/assets/16418547/2396d93f-c7c5-443b-b919-9a94a513dac7)

- just over 1380px and bigger header height:

![Screenshot 2023-10-09 at 11 08 23](https://github.com/apache/arrow/assets/16418547/9364cd5c-34fd-473d-a480-a28c593306ce)

I agree the smaller header size is better in this case also (double navbar), but the logo is too small to read it and that is not something I would keep.

I propose to have the bigger header height for 1400px but in the case of default header height I would use a different logo: [Chevron-only logo](https://arrow.apache.org/img/arrow-logo_chevrons_black-txt_white-bg.png)